spec/fixtures/files/datacite_basic.xml

```diff
@@ -1,21 +1,23 @@
 <?xml version="1.0"?>
-<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://datacite.org/schema/kernel-4" xsi:schemaLocation="http://datacite.org/schema/kernel-4 https://schema.datacite.org/meta/kernel-4.4/metadata.xsd">
-  <identifier identifierType="DOI">10.5072/example-full</identifier>
-  <titles>
-    <title>hello world</title>
-  </titles>
-  <description descriptionType="Other">this is an example description</description>
+<resource xsi:schemaLocation='http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4/metadata.xsd' xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' xmlns='http://datacite.org/schema/kernel-4'>
+  <identifier identifierType='DOI'>10.5072/example-full</identifier>
   <creators>
-    <creator nameType="Personal">
+    <creator>
       <creatorName>Miller, Elizabeth</creatorName>
       <givenName>Elizabeth</givenName>
       <familyName>Miller</familyName>
-      <nameIdentifier schemeURI="https://orcid.org" nameIdentifierScheme="ORCID">1234-5678-9012-1234</nameIdentifier>
+      <nameIdentifier nameIdentifierScheme='ORCID' schemeURI='https://orcid.org'>1234-5678-9012-1234</nameIdentifier>
     </creator>
-    <creator nameType="Organization">
-      <creatorName>Princeton University</creatorName>
+    <creator>
```

Comment:
@hectorcorrea do we no longer allow for organizations?

Reply:
We never really did. Datacite does allow for organizations but we don't allow users to enter them and therefore the code never really produces organizations. This test used to test that how this _could_ work using our custom XML serialization.